server/controllers/pg.js

```diff
@@ -1,6 +1,7 @@
 const { Pool, Client } = require('pg');
 const query = require('.././models/main_functions/queries.js');
-require('dotenv').config();
```

Comment:
Most people I've seen just leave this in the root, I think, so you can have the simpler require (I could be wrong, don't quote me).